### DISCUSSION TITLE
disable notifications

### DISCUSSION THREAD

**Discussion by amigo421** [uphy/obsidian-reminder#144] [Q&A]
Is there an ability to disable popups with notifications?
the panel with list of those is enough for me.

this question is mostly related with mobile version, mobile app is not a running all the time and each time when I open it, some overdue reminders raise a popups, would like to prevent this. in other hand, I don't want to disable the plugin itself , tasks is one of mostly used feature of my knowledge base

**Comment by Superfeuchter** [comment]
The Reminder-Plugin is a very useful tool for my purposes.
But I have the same problem with my notebook. I don't use it so often, and when I open Obsidan after a while (my Vault-Folder is in a Cloud), there are a lot of annoying popups, I have to click away before working.

**Comment by Blaztekk** [comment]
Totallyu agree with you guys, the feature would be really usefull and, i believe, easy to implement. For now im just gonna comment the part of the code...

> Edit : Comment `this.showReminder(reminder);` within `periodicTask()` in `main.js`

**Reply by amigo421** [reply]
could you point the code where to comment this out please?

**Reply by Blaztekk** [reply]
Hi, i commented `this.startPeriodicTask()`; within `this.app.workspace.onLayoutReady()`. Directly on the `**main.js.** `
Its dirty af but working until Owner add the option ;)

**Reply by jjjchens235** [reply]
@Blaz-NanoTeK - thanks for the tip, I tried this, and while it did disable all popups, the reminders no longer showed in the reminder panel. I'm curious if there's a way to disable the popups without removing the reminders altogether from the panel.

**Reply by Blaztekk** [reply]
Comment `this.showReminder(reminder);` within `periodicTask()` @jjjchens235

**Reply by jjjchens235** [reply]
@Blaz-NanoTeK  - that did it, thank you!

**Reply by amigo421** [reply]
for me , it works not as expected (modified version), 
1. now it doesn't group expired items into overdue group but keep those splitted by dates, and now two!! today group there. but it's not so important as point 2: 
2. it doesn't show a popup on click on event in panel. it was useful to re-schedule and quick go to note with the task

I need to suppress a popups on start only, with all other functionality ....
